src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -346,7 +349,154 @@ internal class RecommendationServiceTest : ServiceTestBase() {
+          allOptions = listOf(TextValueOption("NO_RECALL"))
+        ),
+        status = Status.DRAFT,
+        managerRecallDecision = ManagerRecallDecision(
+          selected = ManagerRecallDecisionTypeSelectedValue(
+            value = ManagerRecallDecisionTypeValue.RECALL,
+            details = "Recall"
+          ),
+          allOptions = listOf(
+            TextValueOption(value = "NO_RECALL", text = "Do not recall")
+          ),
+          isSentToDelius = false
+        )
+      )
+      val json = CustomMapper.writeValueAsString(updateRecommendationRequest)
+      val recommendationJsonNode: JsonNode = CustomMapper.readTree(json)
+
+      try {
+        recommendationService.updateRecommendationWithManagerRecallDecision(recommendationJsonNode, 1L, "")
+      } catch (e: InvalidRequestException) {
+        // nothing to do here!!
+      }
+      then(recommendationRepository).shouldHaveNoMoreInteractions()
+    }
+  }
 
+  @Test
+  fun `updates a recommendation with manager recall decision to the database`() {
+    runTest {
+      // given
+      val existingRecommendation = RecommendationEntity(
+        id = 1,
+        data = RecommendationModel(
+          crn = crn,
+          recallConsideredList = listOf(
+            RecallConsidered(
+              createdDate = "2022-11-01T15:22:24.567Z",
+              userName = "Harry",
+              userId = "harry",
+              recallConsideredDetail = "Recall considered"
+            )
+          ),
+          status = Status.RECALL_CONSIDERED,
+          personOnProbation = PersonOnProbation(name = "John Smith"),
+          lastModifiedBy = "Jack",
+          lastModifiedDate = "2022-07-01T15:22:24.567Z",
+          createdBy = "Jack",
+          createdDate = "2022-07-01T15:22:24.567Z",
+          managerRecallDecision = ManagerRecallDecision(
+            selected = ManagerRecallDecisionTypeSelectedValue(
+              value = ManagerRecallDecisionTypeValue.RECALL,
+              details = "Recall"
+            ),
+            allOptions = listOf(
+              TextValueOption(value = "RECALL", text = "Recall")
+            ),
+            isSentToDelius = false,
+            createdBy = "Bill",
+            createdDate = "2022-07-26T09:48:27.443Z"
+          )
+        )
+      )
+
+      // and
+      val updateRecommendationRequest = MrdTestDataBuilder.updateRecommendationWithManagerRecallDecisionRequestData(existingRecommendation)
+
+      // and
+      var recommendationToSave =
+        existingRecommendation.copy(
+          id = existingRecommendation.id,
+          data = RecommendationModel(
+            crn = existingRecommendation.data.crn,
+            managerRecallDecision = updateRecommendationRequest.managerRecallDecision,
+            personOnProbation = updateRecommendationRequest.personOnProbation,
+            recallType = updateRecommendationRequest.recallType,
+            custodyStatus = updateRecommendationRequest.custodyStatus,
```

Comment:
Should this test only be concerned with updating the `managerRecallDecision` rather than all fields?

Reply:
I kept them in to show that the new update doesn't annihilate any pre-existing data and to follow the pattern of using Mockito's `  then(recommendationRepository).should().save(recommendationToSave)` which of course needs all fields.
That said I suppose I could be over testing! I'll make a new commit with a refactor.